src/streams/StreamScreen.js

```diff
@@ -104,6 +115,17 @@ class StreamScreen extends PureComponent<Props> {
             secondary
             onPress={() => delay(this.handleEditSubscribers)}
           />
+          <View style={styles.padding}>
+            <Label style={[styles.marginTop, { fontSize: 16 }]} text="Email Address" />
+          </View>
+          <Touchable onPress={() => this.handleCopy(subscription.email_address)}>
+            <Label
+              style={[
+                { borderColor: BRAND_COLOR, borderWidth: 1.5, borderRadius: 20, padding: 16 },
```

Comment:
The padding and the border-radius seem too much to me. 5px for borderRadius and 8px for the padding can be better.

Reply:
Also, the styles should be applied to `Touchable`, not to `Label`, otherwise, the ripple feedback goes beyond the round borders. I checked, it does get fixed if applied to `Touchable`.

Reply:
@agrawal-d I applied the styles to `Touchable` instead of `Label`, however as you can see in the screenshot below the ripple still goes beyond the round borders. Did you check on Android or iOS device?
<img src="https://user-images.githubusercontent.com/22986282/77795579-cffa1d00-7093-11ea-9050-fe8cf1aa43cc.png" height=450 width=250 />

Also the borderRadius and padding that you suggested seem a bit off from other designs of zulip maybe? What do you say?